domain/src/DomainNegotiatorInterface.php

```diff
@@ -52,6 +52,6 @@ public function negotiateActiveHostname();
   /**
    * Gets the active domain.
    */
-  public function getActiveDomain($rest = FALSE);
+  public function getActiveDomain($reset = FALSE);
```

Comment:
This has already been fixed. Is your fork up-to-date?

https://github.com/agentrickard/domain/blob/8.x-1.x/domain/src/DomainNegotiator.php#L116